test/test_nn.py

```diff
@@ -8490,6 +8490,109 @@ def helper(self, size, groups, memory_format, is_mixed):
                 helper(self, (2, 9, 7, 200, 15), 3, torch.channels_last_3d, is_mixed)
                 helper(self, (2, 60, 7, 200, 15), 3, torch.channels_last_3d, is_mixed)
 
+    @onlyCPU
+    def test_groupnorm_nhwc_cpu(self, device, dtype):
```

Comment:
sorry to nitpick but I think you could just do something like

https://github.com/pytorch/pytorch/blob/2798e8492916f89b432d7461084b0525e0aeffaf/test/test_nn.py#L5527

Reply:
Not a nit pick! I don't work with python all that much so this is helpful :)

Reply:
This is a pytorch specific thing, also it just occurred to me that you would also have to move the test to  the class TestNN rather than TestNNDeviceType (where it currently is) as `TestNNDeviceType` is already parametrized over device

Reply:
This is a pytorch specific thing, also it just occurred to me that you would also have to move the test to the class TestNN rather than TestNNDeviceType (where it currently is) as TestNNDeviceType is already parametrized over device

Reply:
I refactored this to TestNN and tests are passing locally. TYSM